Data_Collection.ipynb

```diff
@@ -0,0 +1,1759 @@
+       "       'valence', 'tempo', 'duration_ms', 'time_signature', 'chorus_hit',\n",
+       "       'sections', 'target', 'bucket'],\n",
+       "      dtype='object')"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "audio_data.columns\n",
+    "\n",
+    "# missing popularity, genre, also LYRICS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "0c98ac33-c40d-4c3d-97af-38759b773ebd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "genre_df = pd.read_csv(\"genre_music.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "b74b8bc9-a76e-4b1b-8c1f-ba2a8198cc84",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>track</th>\n",
+       "      <th>artist</th>\n",
+       "      <th>genre</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Jealous Kind Of Fella</td>\n",
+       "      <td>Garland Green</td>\n",
+       "      <td>edm</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Initials B.B.</td>\n",
+       "      <td>Serge Gainsbourg</td>\n",
+       "      <td>pop</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Melody Twist</td>\n",
+       "      <td>Lord Melody</td>\n",
+       "      <td>pop</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Mi Bomba Sonó</td>\n",
+       "      <td>Celia Cruz</td>\n",
+       "      <td>pop</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Uravu Solla</td>\n",
+       "      <td>P. Susheela</td>\n",
+       "      <td>r&amp;b</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                   track            artist genre\n",
+       "0  Jealous Kind Of Fella     Garland Green   edm\n",
+       "1          Initials B.B.  Serge Gainsbourg   pop\n",
+       "2           Melody Twist       Lord Melody   pop\n",
+       "3          Mi Bomba Sonó        Celia Cruz   pop\n",
```

Comment:
what is the dictionary mapping defined from genre to the integer values in the dataset?

Reply:
didn't do any mapping here, let Preethi/Sana do it in training so they could interpret it in the model!